min-required.txt

```diff
@@ -1,6 +1,6 @@
-lxml==2.1.0  # >=1.3.4 should work
+lxml==3.5.0
```

Comment:
@nijel just to clarify - its the libxml dependencies that im concerned about - not so much the lxml version